packages/components/src/range-control/README.md

```diff
@@ -184,6 +184,23 @@ The maximum value accepted. If higher values are inserted onChange will not be c
 - Type: `Number`
 - Required: No
 
+#### icon
+
+An icon to be shown above the slider next to it's container title.
+
+- Type: `String`
+- Required: No
+- Platform: Only Mobile
+
+#### separatorType
+
+Define if separator line under/above control row should be disabled or full width. By default it is placed below excluding underline the control icon.
+
+- Type: `String Enum`
+- Values: `none` | `fullWidth` | `topFullWidth`
+- Required: No
+- Platform: Only Mobile
```

Comment:
This would be better:

> Platform: Mobile

Let's also add Platform attribute to other props similar to what we did in media-placeholder/README.md
And add these where suitable:

> Platform: Web | Mobile

> Platform: Web

Reply:
Ok. You would like me to do this in current PR or open new one ?

Reply:
Current one would be fine.